Resources/public/js/views/ez-fieldeditview.js

```diff
@@ -229,7 +218,38 @@ YUI.add('ez-fieldeditview', function (Y) {
          * @method initializer
          */
         initializer: function () {
-            Y.eZ.TemplateBasedView.registerPartial('ez_fieldinfo_tooltip', 'fieldinfo-tooltip-ez-partial');
+            Y.eZ.TemplateBasedView.registerPartial('ez_fielddescription_tooltip', 'fielddescription-tooltip-ez-partial');
+            /**
+             * Set if the fieldDefinition description is active or not.
+             *
+             * @property _toggleFieldDefinitionDescription
+             * @protected
+             * @type string
+             * @default 'active'
+             */
+            this._handleFieldDescriptionVisibility = true;
+
+            /**
+             * Set if the fieldDefinition description has the standard display.
+             * Standard means it will use the ez-standard-description CSS rule from fieldedit.css
+             *
+             * @property _useStandardFieldDefinitionDescription
+             * @protected
+             * @type string
+             * @default 'active'
+             */
+            this._useStandardFieldDefinitionDescription = 'true';
```

Comment:
what a confusing choice! really the string `'true'` ? please use a real Boolean or choose others values if a boolean can not represent all the states.